src/main/java/com/extrahardmode/metrics/ConfigPlotter.java

```diff
@@ -79,6 +79,7 @@ public int getValue()
                 {
                     case ALWAYS_ANGRY_PIG_ZOMBIES:
                     case ANIMAL_EXP_NERF:
+                    case ANIMAL_OVERCROWD_CONTROL:
```

Comment:
The reason I only added the boolean because that is how the others were added they skipped over at the percentage given. I may have to also add a option on getMetricsValue to accept Integer values as of now it only accept boolean.